sdks/python/container/base_image_requirements.txt

```diff
@@ -26,6 +26,7 @@
 # TODO(AVRO-2429): Upgrade to >= 1.9.0 only after resolved
 avro-python3==1.8.2
 fastavro==1.0.0.post1
+cloudpickle=2.0.0
```

Comment:
needs == instead of = (see Docker suite failure).

Reply:
aha. That's why it was failing. Done.